apps/site/src/pages/settings/billing/upgrade/upgrade-page-layout.tsx

```diff
@@ -0,0 +1,51 @@
+import { Box } from "@mui/material";
+import { Container } from "@mui/system";
+import { FunctionComponent, ReactNode } from "react";
+
+import {
+  BlockProtocolIcon,
+  BlockProtocolLogoIcon,
+} from "../../../../components/icons";
+
+export const UpgradePageLayout: FunctionComponent<{
+  children?: ReactNode;
+}> = ({ children }) => {
+  return (
+    <Box
+      sx={{
+        minHeight: "100vh",
+        background: "#FAFBFC",
+        display: "flex",
+        flexDirection: "column",
+      }}
+    >
+      <Container
+        sx={{
+          paddingTop: { xs: 6, md: 12 },
+          paddingBottom: { xs: 6, md: 14 },
+          flexGrow: 1,
+        }}
+      >
+        <BlockProtocolLogoIcon
+          sx={{ height: 30, marginBottom: 4, marginLeft: 2 }}
+        />
+        {children}
+      </Container>
+      <Box
+        sx={{
+          py: 5,
+          background: ({ palette }) => palette.common.white,
+          borderTopWidth: 1,
+          borderTopStyle: "solid",
+          borderTopColor: ({ palette }) => palette.gray[20],
+        }}
+      >
+        <Container>
+          <BlockProtocolIcon
+            sx={{ color: ({ palette }) => palette.gray[40], height: 24 }}
+          />
+        </Container>
```

Comment:
The logo is larger than it should be. Please style-match Figma.

Reply:
Which logo, the top logo or the one in the footer?

Reply:
I've adjusted both in https://github.com/blockprotocol/blockprotocol/pull/1152/commits/c8a63b7e89f065f4d03d1638c7765b40d0dd0521, let me know if that's closer to what you had in mind